src/data-display/tooltips/story-helpers.ts

```diff
@@ -0,0 +1,91 @@
+import type { ArgTypes } from '@storybook/addons';
+
+import { POSITIONS } from '@/data-display/tooltips/type';
+
+export const getTooltipArgTypes = (): ArgTypes => ({
+    tag: {
+        name: 'tag',
+        type: { name: 'string' },
+        description: 'root element tag',
+        defaultValue: 'span',
+        table: {
+            type: {
+                summary: 'string',
+            },
+            category: 'props',
+            defaultValue: {
+                summary: 'span',
+            },
+        },
+        control: {
+            type: 'text',
+        },
+    },
+    contents: {
+        name: 'contents',
+        type: { name: 'string' },
+        description: 'Tooltip contents.',
+        defaultValue: 'Tooltip contents',
+        table: {
+            type: {
+                summary: 'string',
+            },
+            category: 'props',
+            defaultValue: {
+                summary: '',
+            },
+        },
+        control: {
+            type: 'text',
+        },
+    },
+    position: {
+        name: 'position',
+        type: { name: 'string' },
+        description: 'Position of tooltip.',
+        defaultValue: 'top',
+        table: {
+            type: {
+                summary: 'string',
+            },
+            category: 'props',
+            defaultValue: {
+                summary: 'top',
+            },
+        },
+        control: {
+            type: 'select',
+            options: [...Object.values(POSITIONS)],
+        },
+    },
+    options: {
+        name: 'options',
+        type: { name: 'object' },
+        description: 'Position of tooltip.',
```

Comment:
Please check if the description is correct

Reply:
thanks!